Project - Chatbot

Looziolooz opened the pull request:
## Netlify link
Add your Netlify link here.
PS. Don't forget to add it in your readme as well.
https://planetagency.netlify.app/

## Collaborators
Add your collaborators here. Write their GitHub usernames in square brackets. If there's more than one, separate them with a comma, like this:
[[github-username-1](https://github.com/Antontall)]

Looziolooz committed:
added html + css code

Looziolooz committed:
added DOM selectors

Looziolooz committed:
added global variables and event listner

Looziolooz committed:
added the entire code

Looziolooz committed:
update file

Looziolooz committed:
added link

HIPPIEKICK commented:
Hi Lorenzo! Did you work together with Anton on this project or not? If yes, please update the PR description with his GitHub username 🙌

Looziolooz commented:
Hello Matilda, yes, Anton worked with me on the project. He only dedicated a few hours (the ideation part, the concept, and even forked without pushing) and then unfortunately didn't get in touch anymore to complete it. So, besides the disappointment, I had to work twice as hard to deliver the project on time. 90% of the project is my work. I'm not sure if I would call it collaboration.

NellaMori commented on code/script.js:
Please remove the console.log() around the code. Generally they need to be deleted when the code has been tested and works as meant to :broom

NellaMori commented on code/script.js:
```diff
@@ -1,53 +1,231 @@
+    confirmQuestion();
+  }
+  else if (currentQuestionNumber === 5) {
+    askForEmail();
```
be careful here, if the user declines do you still wanna ask for the e-mail address? Right not that't the case

NellaMori commented on code/script.js:
```diff
@@ -1,53 +1,231 @@
+  const emailSubmitBtn = document.getElementById('email-submit');
+  emailSubmitBtn.addEventListener('click', () => {
+    const userEmail = document.getElementById('email').value;
+    setTimeout(() => sayThankYou("Thank you for using our service. We've sent more information to your email address.", 'bot'),2000);
```
this function is not being called, check the code to find the bug 🐛

AntonellaMorittu reviewed (changes requested):
Hi Lorenzo and well done completing the chatbot! 🤖 It looks quite good and works almost perfectly, I left a couple of comments for you to review the app behaviour and fix the little bug there is now when declining or confirming. To improve even more the UX you could add validation for the name field so that is not empty for example? Thank you ⭐ almost there 💪

Looziolooz committed:
changed the last part of the conversation

Looziolooz committed:
added return message in the handleNameInput

Looziolooz committed:
update code

Looziolooz commented:
Hi Antonella,
I took the time to review the code and make the required changes. Sorry for the delay but I had to work five days in a row. As for the chatbot project, I removed the part regarding the email address request. I inserted a new thank you message and removed console.log that I forgot to remove. I saw that there is still a problem regarding the username. If it remains empty I have inserted a new message from the bot, it still remains incomplete. Since I have little time, I would like to dedicate the remaining time to the project to be presented tomorrow. I hope it goes well. Thanks for your patience. Ciao!!
